File: level84/classwork/codwars4.js
//https://www.codewars.com/kata/51e704f2d8dbace389000279/train/javascript
function arraysSimilar(arr1, arr2) {
  arr1 = arr1.sort((a, b) => a - b);
    arr2 = arr2.sort((a, b) => a - b);
    if (arr1.length != arr2.length) {
        return false;
    } else {
        for (let i = 0; i < arr1.length; i++) {
            if (arr1[i] !== arr2[i]) {
                return false;
            }
        };
        return true;
    };
}

File: level80/homework/codwars4.py
#https://www.codewars.com/kata/51e704f2d8dbace389000279/train/python
def arrays_similar(arr1, arr2):
    if len(arr1) != len(arr2):
        return False
    
    dict1 = {}
    dict2 = {}
    
    for elem in arr1:
        dict1[elem] = dict1.get(elem, 0) + 1
    
    for elem in arr2:
        dict2[elem] = dict2.get(elem, 0) + 1
    
    return dict1 == dict2